library/aesce.c

```diff
@@ -30,19 +52,19 @@
 #   if __clang_major__ < 4
 #       error "A more recent Clang is required for MBEDTLS_AESCE_C"
 #   endif
+#        pragma clang attribute push (__attribute__((target("crypto"))), apply_to=function)
```

Comment:
Above the define guard sets it that clang_version should be > 3

```
if defined(__aarch64__) && !defined(__ARM_FEATURE_CRYPTO) && \
    defined(__clang__) && __clang_major__ > 3
```

But here the attribute is being pushed only for `clang > 4` . What happens in between? Should we bump the version above?

Reply:
AIUI the check above is that clang version > 3, whereas here it's set if clang version is NOT < 4, ie, if clang version >= 4. So they are consistent. But I agree it would be clearer to use the same form of the expression in both places (i.e., use `>= 4` above instead of `> 3`).